spec/spec_helper.rb

```diff
@@ -19,6 +19,13 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 
+require File.expand_path("../../config/environment", __FILE__)
+require "rails/all"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.